ui/package.json

```diff
@@ -102,12 +102,12 @@
     "react-calendar": "^2.18.1",
     "react-dom": "^16.6.3",
     "react-iframe": "^1.8.0",
-    "react-interactive": "^0.8.1",
+    "react-interactive": "^0.9.1",
```

Comment:
My shallow understanding of semvar is that `^` means any version up till the next major version will be pulled. Wouldn't that mean `0.9.1` should have been pulled in when we specify `^0.8.1`?

Reply:
Nearly -- the ^caret ranges are specified as follows:

> Allow changes that do not modify the first non-zero digit in the version, either the 3 in 3.1.4 or the 4 in 0.4.2.

https://yarnpkg.com/en/docs/dependency-versions#toc-caret-ranges